### PR TITLE
feat: New singlestore quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

Added a new quickstart that utilizes an example OTeL tool collector to get metrics from SingleStore's API and ingest it into NR.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/singlestore/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description: |
+  ## About this integration
+    This integration uses an OpenTelemetry(OTeL) collector to gather metrics from [SingleStore's management API](https://docs.singlestore.com/cloud/reference/management-api/reference/) and visualize them in New Relic.
+
+  ## What does this integration cover?
+    This integration gives you complete visibility into your resource utilization and workload statistics for your Singlestore Database.
+
+summary: |
+  Monitor the performance of your SingleStore database using OTeL.
+icon: logo.svg
```
Comment (RamanaReddy8801) [inline quickstarts/singlestore/config.yml:11]: Logo will bot visible under white background, kindly update logo

**Review by RamanaReddy8801 — CHANGES_REQUESTED** [review]

File: quickstarts/singlestore/config.yml (hunk 1 above)
Comment (jcountsNR) [reply quickstarts/singlestore/config.yml]: @RamanaReddy8801 , I swapped it out for one with a clear background. Let me know if that works!

Comment (RamanaReddy8801) [reply quickstarts/singlestore/config.yml]: @jcountsNR, 
That's fine, please make the change in the datasource folder as well.

Comment (jcountsNR) [reply quickstarts/singlestore/config.yml]: @RamanaReddy8801 done, let me know if there is anything else missing here.

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM!

**Review by pkudikyala — APPROVED** [review]
Looks good to me.

**Pull request merged**